com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanel.cs

```diff
@@ -145,20 +122,20 @@ public struct TextFont {
         
         static bool _ShowDebugLog = false;
         
-        protected virtual void loadConfiguration() {}
+        protected virtual void onEnable() {}
 
-        Dictionary<string,UIWidgetsPanel.TextFont> _internalTextFonts = new Dictionary<string, UIWidgetsPanel.TextFont>();
+        Dictionary<string,TextFont> _internalTextFonts = new Dictionary<string, TextFont>();
         protected void AddFont(string family, List<string> assets, List<int> weights) {
             if (assets.Count != weights.Count) {
-                UnityEngine.Debug.Log($"The size of {family}‘s assets should be equal to the weights'.");
+                UnityEngine.Debug.LogError($"The size of {family}‘s assets should be equal to the weights'.");
                 return;
             }
             if (!_internalTextFonts.ContainsKey(family)) {
```

Comment:
please keep in mind:
(1) if there are same code blocks in different place, try to make it in one function
(2) if two API look to be the same (e.g., the two AddFont functions in UIWidgetsPanel and UIWidgetsEditorPanel), make their functionality the same. otherwise user will be confused when they call the same API in different places with different outcomes